README.windows

```diff
@@ -0,0 +1,59 @@
+Windows support for gominer requires very specific build steps.  Unlike Linux
```

Comment:
Don't know yet!  Hope we can make this environment work with opencl as well so that we don't need multiple build machines .